CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ if(NOT COMPILER_SUPPORTS_CXX11)
 endif()
 
 option(HIGHFIVE_UNIT_TESTS "enable unit tests" TRUE)
-option(USE_BOOST "enable Boost Support" TRUE)
+option(USE_BOOST "enable Boost Support" FALSE)
```

Comment:
Having USE_BOOT false and HIGHFIVE_UNIT_TESTS true does not make sense as to have the unit tests you need boost. Should that also be changed?